homeassistant/components/media_player/plex.py

```diff
@@ -750,17 +750,17 @@ def play_media(self, media_type, media_id, **kwargs):
 
         media = None
         if media_type == 'MUSIC':
-            media = self.device.server.library.section(
+            media = self.device._server.library.section(
```

Comment:
Anything starting with `_` is an internal attribute and can change at any time as it's not part of their public API. We should not use it. Also, this code was working?

Reply:
Regarding the "_", see my comment there:
https://github.com/home-assistant/home-assistant/issues/19623#issuecomment-450464838
Basically their API uses "_" for static states (Server, client name) and plain attributes for data that should be reloaded before returned.  I don't believe that they are subscribing to the convention that "_" is a private attribute.

device._server is the only way to retrieve the _server attribute using their API. We could store a server attribute internally to get around typing "_". The problem with that is we still need to access server attribute "_baseurl" so we can't get rid of all underscores.

I have explained my rationale for this hotfix in my comment above. The fix resumes the PlayMedia capabilities of the Plex plugin and does not do any other refactoring that can break other parts. 

I am new to HomeAssistant so I cannot confirm that this ever worked but I assume the plugin was tested by its author. From my bug report, it appears like "server" was/(should be) available but the source code says otherwise. Fixing upstream might be another, albeit slower, possibility.